### ISSUE TITLE
Failed to fetch error running blazor pages

### ISSUE THREAD

**Issue by HoughAPH** [Netcode-Hub/DemoCustomJwtDotNet8BlazorWasmAuthAndAuthWithIdentityManagerSolution#1]
Hi
I hope you can help.
Running the code as is with the configured port numbers is  a problem for me.
Am I supposed to be able to set the BlazorWebAssemblyApp as the startup project and do a login using the API?
This always gives an error "Failed to fetch" error.
I tried changing the port numbers.  Say 7202 for the API and the blazor pages.  This gives and error of Post Method not allowed.

I tried running in IE Express  and https.

If I set the API as the startup project, I can login using the swagger successfully.

The problem is with CORS I think.
If you have any advice I will be most grateful.

THANKS for this example on .net 8!
I did learn a lot from it.

Thanks in advance for any additional advice.
Regards

Andre

**Comment by Netcode-Hub** [comment]
Hy, sorry for having this error. Your solution awaits you here …
Enable Cross-Origin Resource Sharing (CORS) for Seamless Client-Server
Interaction in .NET 8 Web API https://youtu.be/VCcaqR3lS-U
Check it out

**Comment by HoughAPH** [comment]
Hi

Wow thanks a million!

I downloaded the repository from the video you proposed.
I had to tweak it a little bit to make it work, but it helped me tremendously,.
I still get two errors in the console dev tools.  Not too critical I think.  
Maybe my environment setup.  


![Screenshot BalzorApp error](https://github.com/Netcode-Hub/DemoCustomJwtDotNet8BlazorWasmAuthAndAuthWithIdentityManagerSolution/assets/50958705/3004099d-c995-42f6-a68a-8927f577ac39)

That link contains this url:
http://localhost:56699/674663d26e204b01a5355652cc03c849/browserLinkSignalR/negotiate?requestUrl=https%3A%2F%2Flocalhost%3A7189%2F&browserName=&userAgent=Mozilla%2F5.0+(Windows+NT+10.0%3B+Win64%3B+x64)+AppleWebKit%2F537.36+(KHTML%2C+like+Gecko)+Chrome%2F124.0.0.0+Safari%2F537.36&browserIdKey=window.browserLink.initializationData.browserId&browserId=f7d4-1f3b&clientProtocol=1.3&_=1716389631417

SignalR?
I have tight add blockers on my chrome.  Just guessing.

I'm looking for a new .NET developer job.
 My frontend dev is not as good as my backend dev.
 So I'm doing these tutorials to use a web app to consume APIs that I have developed before.
 While upgrading them to .NET 8 etc.

Thanks again for your help.
I subscribed to your channel.  
One of the very best on YouTube.

Kind regards
Andre
